sm/src/objects.mk

```diff
@@ -36,7 +36,7 @@ keystone-sm-sources += crypto.c sha3/sha3.c ed25519/fe.c ed25519/ge.c ed25519/ke
                         hmac_sha3/hmac_sha3.c
 
 # Platform
-keystone-sm-sources += platform/$(PLATFORM)/platform.c
+keystone-sm-sources += platform/$(KEYSTONE_PLATFORM)/platform.c
```

Comment:
This should be unnecessary once you've implemented the `BR2_TARGET_OPENSBI_PLAT="unmatched"` change